src/app/submission/Database.ts

```diff
@@ -20,9 +20,9 @@ const submissionTableItemSchema = z.object({
   pk: dynamoDBStringSchema,
   sk: dynamoDBStringSchema,
   pdfKey: dynamoDBStringSchema,
-  dateSubmitted: dynamoDBStringSchema,
-  formName: dynamoDBStringSchema,
-  formTitle: dynamoDBStringSchema,
+  dateSubmitted: dynamoDBStringSchema.optional(),
```

Comment:
Had je deze niet zo omgebouwd dat alle nieuwe entries altijd deze optionele velden hebben?

Reply:
Ja, maar een paar (dubieuze, door ons ingediende) hebben dat niet. In prod zijn het er geloof ik geen enkele, in accp een stuk of 3. Ben zelf ook beetje in dubio wat ermee te doen, maar de praktijk nu is dat ze niet gegarandeerd gevuld zijn.